site/website/blog/2020-09-18-cdc-meets-stream-processing.md

```diff
@@ -0,0 +1,178 @@
+
+Jet doesn't delegate its cluster management and fault tolerance concerns
+to an outside system like ZooKeeper. It reuses the groundwork
+implemented for Hazelcast IMDG: cluster management and the IMap, and
+adds its own implementation of Chandy-Lamport distributed snapshots. If
+a cluster member fails, Jet will restart the job on the remaining
+members, restore the state of processing from the last snapshot, and
+then seamlessly continue from that point. For further details, consult
+our [documentation on the
+topic](https://jet-start.sh/docs/next/architecture/fault-tolerance).
+
+Extending this functionality umbrella to cover Debezium has been
+surprisingly simple. All that was really needed was to add Debezium’s
+source offset to Jet’s snapshots. This way, whenever Jet needs to
+execute a recovery, it can also pass the recovered offset to Debezium,
+which in turn is able to resume the data flow from that offset.
+
+One other thing we did and might be worth mentioning is that the Jet
+integration also makes use of Debezium’s [new record state
+extraction](https://debezium.io/documentation/reference/1.2/configuration/event-flattening.html)
+SMT (Simple Message Transformation), for the purpose of message
+structure simplification. With this transformation in effect, only the
+"after" structure of the Debezium event envelope is processed by Jet.
+However, whether this is a good idea or not, only time will tell. I
+personally think that if and when we will start covering schema changes
+more, we might end up re-enabling the full Debezium event content.
+
+## Examples
+
+The simplest example of using the Jet-Debezium integration would be our
+[CDC tutorial](https://jet-start.sh/docs/next/tutorials/cdc) that I’ve
+already mentioned above. A more involved one can be seen in my
+colleague’s, Nicolas Fränkel’s [blog
+post](https://jet-start.sh/blog/2020/07/16/designing-evergreen-cache-cdc).
+
+## License
+
+The Jet - Debezium integration is currently provided under the [Apache
+License, Version 2](https://www.apache.org/licenses/LICENSE-2.0.txt),
+just like most of Jet and Debezium, so making full usage of the
+combination of the two should have no impediments in your own projects.
+
+## Looking ahead
+
+At the moment of writing the Jet-Debezium integration is fully finished
+only for MySQL and Postgres databases and has been [released in version
+4.2](https://jet-start.sh/blog/2020/07/14/jet-42-is-released) of Jet.
+When we will cover further connectors and if we will extend existing
+ones (for example by adding handling for database schema changes),
+remains to be seen.
+
+The functionality provided by Debezium, the ability to allow modern
+processing of legacy data is a great fit to Jet’s ability to carry out
+that processing efficiently. The combination of the two has the
+potential to become much more than the sum of their parts. I’m very much
+looking forward to finding out what this merger can lead to. Stay tuned!
```

Comment:
Doesn't `merger` imply some cooperation between us and Debezium?

Maybe use something more abstract, like `synergy` or whatever fancy word you can think of.

Reply:
Hook-up? ;)

Reply:
"integration" maybe